### PR TITLE
4.5.5 - Remove Jenkins and replace with GitHub Actions

### PR DESCRIPTION
**Pull request by Wolftousen**
As the title says.  It also enhances the exercises a bit and links them up with the upcoming GitHub chapter

### CODE REVIEW

File: docs/4-software-development-practices/4.5.5-test-automation.md
Comment (jburns24) [inline docs/4-software-development-practices/4.5.5-test-automation.md:21]: I would remove reference to the `GitHub Actions` section of the bootcamp as I am not sure when that initiative will get finished or added. The bootcamp as it is is a rather gross offender of saying "do x" without introduction to "x". I am ok if we do that in this situation

File: docs/4-software-development-practices/4.5.5-test-automation.md
Comment (jburns24) [inline docs/4-software-development-practices/4.5.5-test-automation.md:29]: Assuming this wording is based on the assumption that the GitHub Actions new section will be finished and merged before you section. If so let's not assume that. 

If it is not based on that assumption I must have a missunderstanding

File: .cspell.json
Comment (jburns24) [inline .cspell.json:13]:
```suggestion
```

NP: Should not actually be required with the bug in this file fixed

**Review by jburns24 — APPROVED** [review]

**Comment by jburns24** [comment]
Want to hold off on merging this until @rhoofard wraps up the GitHub chapter

**Comment by amber-beasley-liatrio** [comment]
@Wolftousen , I saw you wanted to wait until the GitHub chapters were done before merging your changes and I saw you had references to the GitHub chapter.

I also wanted to piggy back off of the GitHub chapter so I can reference any sample/getting started project a camper has already created.

There is a bit of a continuity problem. The chapter we are working on are in 4 but GitHub is in Chapter 9 

image

So they would read our chapters first before getting to the GitHub chapter. 

@rhoofard (maybe we can sync up about order of the chapters?)

**Comment by jburns24** [comment]
@amber-beasley-liatrio When I touched bases with Ryan this morning I mentioned that I wanted this moved to Chapter 2. I think he has been swamped with BlueYonder work and hasnt touched this in few days. But he is aware that it should be moved to chapter 2.

**Pull request merged**
